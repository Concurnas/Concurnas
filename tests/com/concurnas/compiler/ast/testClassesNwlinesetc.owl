

//define classes, interfaces et al

def some() void
{
	g()
}

d=s;
class MyClass<X>(-n String, -m String)
{
	a =6;
	f=6;
	def add(toAdd X) void	{
	d()
		}
	
	def do(n int) void
	{
		this.a= n**5
			}
	
	p=4
	
	ompa = def () int { return 5 
	}
	
	ompa = def () int { return 5	} 
	
	l= 5
	
	
	l=0 ; l=r()
	
	class inner{
	}
	
}

class Singleton
{
}

if(a> 5)
{

}

else
{}

while(false){
a=4; g=y

}


while(false){
a=4; g=y 
}

mmop = { 1->2, 4
-> 5}

a = [1,2 
,3 ]


with(logging)
{
log("");
}

zzz = new A.C(2,3);

bobby List<Integer> = new List<Integer>(1);

a = actor MyClass<String>(3);