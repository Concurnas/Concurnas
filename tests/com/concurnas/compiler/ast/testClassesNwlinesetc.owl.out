{
def some ( ) void {
g ( ) ;
}
d = s ;
class MyClass < X > ( protected -n String , protected -m String ) {
a = 6 ;
f = 6 ;
def add ( var toAdd X ) void {
d ( ) ;
}
def do ( var n int ) void {
this . a = n ** 5 ;
}
p = 4 ;
ompa = def ( ) int {
return 5 ;
}
;
ompa = def ( ) int {
return 5 ;
}
;
l = 5 ;
l = 0 ;
l = r ( ) ;
class inner {
}
}
class Singleton {
}
if ( a > 5 ) {
}
else {
}
while ( false ) {
a = 4 ;
g = y ;
}
while ( false ) {
a = 4 ;
g = y ;
}
mmop = { 1 -> 2 , 4 -> 5 } ;
a = [ 1 , 2 , 3 ] ;
with ( logging ) {
log ( "  " ) ;
}
zzz = new A.C ( 2 , 3 ) ;
bobby List < Integer > = new List < Integer > ( 1 ) ;
a = new com.concurnas.lang.TypedActor < MyClass < String > > ( 3 ) ;
}